imap_processing/mag/l1a/mag_l1a.py

```diff
@@ -139,6 +148,8 @@ def process_packets(
         primary_start_time = TimeTuple(mag_l0.PRI_COARSETM, mag_l0.PRI_FNTM)
         secondary_start_time = TimeTuple(mag_l0.SEC_COARSETM, mag_l0.SEC_FNTM)
 
+        mago_is_primary = mag_l0.PRI_SENS == 0
```

Comment:
This 0 is a bit magical. I see you added some other Enums for datamode, would you want to add another Enum for the primary sensor flag here as well?